website/docs/introduction/getting_started.mdx

```diff
@@ -72,8 +72,19 @@ Alternatively, you can manually add the dependency to your app from within your
 Then, install packages with `flutter pub get`.
 
 <When codegen={true}>
+  ## Generating code from the terminal. 
   You can now run the code-generator with{" "}
   <code>flutter pub run build_runner watch</code>.
```

Comment:
What about changing the `<code>` into a ```` ``` ````?

The label doesn't work, and I don't like the idea of adding semi-redundant information

````suggestion
  Then, you'll need to start the code-generator. To do so, run the following command in a terminal:
  ```
  flutter pub run build_runner watch</code>.
  ```
````